src/utils/format.ts

```diff
@@ -123,3 +123,27 @@ export const retrieveDirectorNameFromFiling = (filing: OfficerFiling ): string =
     return "";
   }
 }
+
+export const lowerCaseAllWordsExceptFirstLetters = (country: string | undefined) => {
+  if (!country){
+    return "";
+  }
+
+  const wordsForAllLowerCase = ["AND", "OF", "THE", "DA", "PART", "CITIZEN"];
```

Comment:
It feels like this should be config, may need to be expanded in the future if new countries are added or renamed.

Reply:
Improving on what ROE used, go for it and move to global constants.

Reply:
List moved to constants